README.rst

```diff
@@ -30,192 +30,8 @@
-  for detailed support information for each framework.
-
-* `Concurrent model execution support
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/model_configuration.html#instance-groups>`_. Multiple
-  models (or multiple instances of the same model) can run
-  simultaneously on the same GPU.
-
-* Batching support. For models that support batching, the server can
-  accept requests for a batch of inputs and respond with the
-  corresponding batch of outputs. The inference server also supports
-  multiple `scheduling and batching
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/model_configuration.html#scheduling-and-batching>`_
-  algorithms that combine individual inference requests together to
-  improve inference throughput. These scheduling and batching
-  decisions are transparent to the client requesting inference.
-
-* `Custom backend support
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/model_repository.html#custom-backends>`_. The inference server
-  allows individual models to be implemented with custom backends
-  instead of by a deep-learning framework. With a custom backend a
-  model can implement any logic desired, while still benefiting from
-  the GPU support, concurrent execution, dynamic batching and other
-  features provided by the server.
-
-* `Ensemble support
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/models_and_schedulers.html#ensemble-models>`_. An
-  ensemble represents a pipeline of one or more models and the
-  connection of input and output tensors between those models. A
-  single inference request to an ensemble will trigger the execution
-  of the entire pipeline.
-
-* Multi-GPU support. The server can distribute inferencing across all
-  system GPUs.
-
-* The inference server provides `multiple modes for model management
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/model_management.html>`_. These
-  model management modes allow for both implicit and explicit loading
-  and unloading of models without requiring a server restart.
-
-* `Model repositories
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/model_repository.html#>`_
-  may reside on a locally accessible file system (e.g. NFS) or in
-  Google Cloud Storage.
-
-* Readiness and liveness `health endpoints
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/http_grpc_api.html#health>`_
-  suitable for any orchestration or deployment framework, such as
-  Kubernetes.
-
-* `Metrics
-  <https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/metrics.html>`_
-  indicating GPU utilization, server throughput, and server latency.
-
-.. overview-end-marker-do-not-remove
```

Comment:
You shouldn't delete the end marker or the license at the end. Notice how the README doesn't look right on github.

Reply:
I see. I can add that back in. I deleted too much based on the PR you pointed to.

Reply:
The PR doesn't delete those things. Perhaps looking at the wrong PR?

Reply:
You are correct, it doesn't. It was purely my mistake. I saw large chunks deleted and assume (incorrectly) that everything was deleted. I added back the end marker and license at the end so now it matches the PR in the example.